kubernetes/platform-services/values.yaml

```diff
@@ -313,3 +315,74 @@ hypertrace-ui:
 
 kafka-topic-creator:
   enabled: false
+
+hypertrace-alert-engine: 
+  hypertraceAlertEngineConfig:
+    queryService:
+      host: *queryServiceHost
+      port: *queryServicePort
+    attributeService:
+      host: *attributeServiceHost
+      port: *attributeServicePort
+  notificationChannelsConfig:
+    channels:
+      [
+        {
+          "channelName": "channel-1",
+          "channelId": "channel-id-1",
+          "channelConfig": [
+            {
+              "channelConfigType": "WEBHOOK",
+              "url": "<override slack url>",
+              "webhookFormat": "WEBHOOK_FORMAT_SLACK"
+            }
+          ]
+        }
+      ] 
+  alertRulesConfig:
+    rules:
+      [
+        {
+          "id": "notification_rule_1",
+          "ruleName": "high_avg_latency",
+          "description": "Alert for high avg latency of payment service",
+          "eventConditionId": "event_condition_1",
+          "eventConditionType": "MetricAnomalyEventCondition",
+          "channelId": "channel-id-1",
+          "eventCondition": {
+            "metricSelection": {
+              "metricAttribute": {
+                "key": "duration",
+                "scope": "SERVICE"
+              },
+              "metricAggregationFunction": "METRIC_AGGREGATION_FUNCTION_TYPE_AVG",
+              "metricAggregationInterval": "PT15S",
+              "filter": {
+                "leafFilter": {
+                  "lhsExpression": {
+                    "attribute": {
+                      "key": "name",
+                      "scope": "SERVICE"
+                    }
+                  },
+                  "valueOperator": "VALUE_OPERATOR_EQ",
+                  "rhsExpression": {
+                    "stringValue": "customer"
```

Comment:
This seems specifically tied to hotrod. Can we make it more abstract?

Reply:
this is just some starter stuff, can be generalised later